### PR TITLE
Switched from csso to clean-css 

### PR DESCRIPTION
**Pull request by MattDiMu**
PR as requested in #324 

Set the default css-minification value to ie8-support. See [here](https://github.com/jakubpawlowicz/clean-css#how-to-set-compatibility-mode) for more details.

I've searched all recipes as well and hopefully haven't forgot anything :)

Regards
Matt

### CODE REVIEW

**Comment by silvenon** [comment]
Thanks a bunch! Could you squash your commits into one? Also, we don't really care for older IEs, so you can drop the IE8 compatibility part, we'll let users decide to add it if necessary.

**Comment by silvenon** [comment]
Ping.

**Comment by MattDiMu** [comment]
First time I've squashed commits - hopefully I did it right :)

Removed the compatibility in the gulpfile, but remained it in the recipies (for users who need it).

**Comment by kevva** [comment]
This needs a rebase too :).

File: app/templates/gulpfile.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ gulp.task('html', ['styles'], function () {
   return gulp.src('app/*.html')
     .pipe(assets)
     .pipe($.if('*.js', $.uglify()))
-    .pipe($.if('*.css', $.csso()))
+    .pipe($.if('*.css',$.minifyCss({compatibility: '*'})))
```
Comment (appleboy) [inline app/templates/gulpfile.js]: need one space

```
.pipe($.if('*.css', $.minifyCss({compatibility: '*'})))
```

**Comment by kevva** [comment]
@MattDiMu, why did you close?

**Comment by silvenon** [comment]
Maybe you misunderstood what "rebase" means. [This](http://mislav.uniqpath.com/2013/02/merge-vs-rebase/) blog post might help with understanding these concepts a bit.

**Pull request closed without merging**